assessment/assessment.py

```diff
@@ -337,13 +338,20 @@ def GET(self, bank_id):
     @utilities.format_response
     def POST(self, bank_id):
         try:
+            local_data_map = self.data()
             am = autils.get_assessment_manager()
             bank = am.get_bank(utilities.clean_id(bank_id))
             bank.use_isolated_bank_view()
 
             form = bank.get_assessment_form_for_create([SIMPLE_SEQUENCE_ASSESSMENT])
 
-            form = utilities.set_form_basics(form, self.data())
+            form = utilities.set_form_basics(form, local_data_map)
+
+            if 'levelId' in local_data_map:
+                form.level = utilities.clean_id(local_data_map['levelId'])
```

Comment:
@xaviaracil Tests for these two fields (`levelId` and `rubricId` would also be good). Thanks!